x/market/handler_test.go

```diff
@@ -85,10 +87,44 @@ func TestHandlerMsgSwapTooSmall(t *testing.T) {
 	require.False(t, res.IsOK(), "expected failed message execution: %v", res.Log)
 
 	// Reset oracle price
-	input.oracleKeeper.SetLunaSwapRate(input.ctx, askDenom, askLunaPrice)
 	askLunaPrice = sdk.NewDecWithPrec(1000, 1)
 	input.oracleKeeper.SetLunaSwapRate(input.ctx, askDenom, askLunaPrice)
 
 	res = handler(input.ctx, msg)
 	require.True(t, res.IsOK(), "expected successful message execution: %v", res.Log)
 }
+
+func TestHandlerExceedDailySwapLimit(t *testing.T) {
+	input := createTestInput(t)
+	handler := NewHandler(input.marketKeeper)
+
+	offerCoin := sdk.NewInt64Coin(assets.MicroSDRDenom, 100)
+
+	// Set oracle price
+	offerLunaPrice := sdk.NewDec(1)
+	input.oracleKeeper.SetLunaSwapRate(input.ctx, offerCoin.Denom, offerLunaPrice)
+
+	// Day 0 ... trade goes through, even though Luna doesn't even have a stated issuance.
```

Comment:
It conflict with the comment at x/market/handler.go:32.

Reply:
As commented, the new comment on handler.go:32 should be consistent with this one.